agent/functional_tests/tests/functionaltests_unix_test.go

```diff
@@ -567,7 +567,8 @@ func TestFluentdTag(t *testing.T) {
 	// tag was added in docker 1.9.0
 	RequireDockerVersion(t, ">=1.9.0")
 
-	if runtime.GOOS == "arm64" {
+	// Skipping the test for arm as they do not have official support for Arm images
+	if runtime.GOARCH == "arm64" {
 		t.Skip()
```

Comment:
Mind adding a string to the output with `t.Skip("unsupported, no image for arm64")` for posterity?